### PR TITLE
update to PROJ.4 6.0.0

### PR DESCRIPTION
**Pull request by snowman2**
In an effort to be able to release when 6.0.0 is ready, attempting to prepare build with 6.0.0RC4.

### CODE REVIEW

**Comment by snowman2** [comment]
@conda-forge-admin please rerender.

**Comment by snowman2** [comment]
@conda-forge-admin please rerender.

**Comment by snowman2** [comment]
The error seems to be:
```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {'gxx_linux-64}}'}
```
Not sure why this is an issue.

**Comment by snowman2** [comment]
@ocefpaf, do you see any obvious issues?

**Comment by snowman2** [comment]
Windows 2008 compilers fail because `Must have C++11 or newer` as that is required by PROJ.4 6.0.0.

**Comment by snowman2** [comment]
@ocefpaf, looks like it is passing - windows is a partial pass.

**Comment by ocefpaf** [comment]
I guess we can skip `win_c_compilervs2008cxx_compilervs2008` it does not look like it is worth patching, right?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,43 @@
-{% set version = "5.2.0" %}
-{% set datumgrid_ver = "1.7" %}
+{% set version = "6.0.0" %}
+{% set datumgrid_ver = "1.8" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml:2]: We probably need to split the `datumgrid` part into its own package, but let's do that in another PR.

Comment (snowman2) [reply recipe/meta.yaml:2]: Sounds like a good idea 👍

**Comment by ocefpaf** [comment]
@snowman2 awesome PR! Thanks a lot for taking care of this. Do you want to add yourself as a maintainer? (You can do that in another PR with `[skip ci]`).

**Comment by snowman2** [comment]
> Do you want to add yourself as a maintainer?

Sure, just a moment.
